public/locales/en/learnComponents.json

```diff
@@ -86,5 +86,43 @@
     "testifyInWriting": "Testify in writing",
     "testifyOrally": "Testify orally",
     "writeOrCall": "Write or call them"
+  },
+  "legislative": {
+    "title": "Understanding the Massachusetts Legislative Process",
+    "intro": "Some of the key steps in the legislative process for how most bills become laws in MA.",
+    "content": [
+      {
+        "title": "Filing",
+        "paragraph": "A Representative, Senator, or the Governor must file a bill. Constituents can write to their legislators (typically their own  district's House or Senate member) to propose a bill to be filed. This can be done at any time, but is typically done before the start of the legislative session (i.e. before the third Friday in January of odd-numbered years). Bills filed later, during the legislative session, require special approval."
+      },
+      {
+        "title": "Testimony",
+        "paragraph": "All stakeholders have the chance to share their thoughts with the legislature by submitting written testimony. Typically, testimony is submitted to the legislative Committee responsible for the bill in advance of their public hearing. You can also deliver oral testimony by attending a hearing or you can reach out to your legislators and speak to them directly."
+      },
+      {
+        "title": "Public hearing",
```

Comment:
np: Should we make it so that each word is capitalized under the titles?

Reply:
I was trying to keep it consistent with the existing text, but honestly, agree 100% - going to make the change